server/lib/file-upload/local-filesystem.js

```diff
@@ -6,9 +6,7 @@ import path from 'path'
 import thenify from 'thenify'
 import util from 'util'
 
-// How long browsers can cache resources for (in milliseconds). These resources should all be pretty
-// static, so this can be a long time
-const FILE_MAX_AGE_MS = 14 * 24 * 60 * 60 * 1000
+import { FILE_MAX_AGE_MS } from './index.js'
```

Comment:
This feels like a pretty weird import to me (I'd expect imports to go in the other direction), maybe this constant should be in a different file?